deno/lib/helpers/errorUtil.ts

```diff
@@ -1,5 +1,7 @@
 export type ErrMessage = string | { message?: string };
 export const errToObj = (message?: ErrMessage) =>
   typeof message === "string" ? { message } : message || {};
+
+// biome-ignore lint/suspicious/noShadowRestrictedNames: <explanation>
 export const toString = (message?: ErrMessage): string | undefined =>
```

Comment:
Might be worth renaming to `errToString` for symmetry, and to avoid shadowing.

Reply:
That'll be a breaking change so I'll put that in it's own commit for the sake of cleanliness